composer.json

```diff
@@ -9,12 +9,17 @@
         "twig/twig": "^1.34"
     },
     "require-dev": {
-        "openeuropa/code-review": "~1.0",
+        "cweagans/composer-patches": "~1.6",
+        "openeuropa/code-review": "~1.0-beta2",
```

Comment:
Is it the same of `"openeuropa/code-review": "~1.0.0-beta2"`

Reply:
Yep, that's an old habit, sorry. I'll change it.